### ISSUE TITLE
default image tag for helmOperator in Helm?

### ISSUE THREAD

**Issue by chy168** [weaveworks/flux#1222]
hi,

I am a new comer to Flux. I install flux via `helm install` and ran into a `ssh looping problem`, the operator keep saying the git repo not ready. The most close description is related to ed09d7b8348d429c4fed13f27dbfb6f1de810794. So I assume my `git.url` might be wrong format, I keep trying every kind of url format as i can but in vain.

After many trial and error, I found the ssh fix might not included in my helm operator deployment due to the default `helmOperator.tag` of Flux Helm is point to `0.1.0-alpha`.
Finally got it work via:

```
helm install --name flux \
--set helmOperator.create=true \
--set git.url=ssh://git@github.com/chy168/flux-helm-test.git \
--set git.chartsPath=charts \
--set image.tag=1.4.2 \
--set helmOperator.tag=master-a1e23c8 \
--namespace flux \
weaveworks/flux
```

I set `helmOperator` tag to the latest tag via `--helmOperator.tag=master-a1e23c8` at installation and it works like a charm.

i would like to suggest if we could change the default `helmOperator.tag` to another or just build `weaveworks / helm-operator` with `latest` tag (there is not `latest` tag right now) and always point to? (not sure if that is a ideal way)

Thank you!

**Comment by squaremo** [comment]
Hi there, we released a quay.io/weaveworks/helm-operator:0.1.1-alpha not long ago, and I've updated the examples and Helm chart to use that new version. Sorry for the docs and examples mismatch in the meantime :-S

**Comment by chy168** [comment]
i agree `:latest` is not a good idea because flux is moving fast, it might cause more problems for new comer (like me). 

thank you @squaremo 😺
